.rubocop.yml

```diff
@@ -1,5 +1,4 @@
 require:
-  - rubocop-rails
```

Comment:
This was causing an issue similar to what this person ran into:
https://github.com/codeclimate/codeclimate-rubocop/issues/178